Makefile

```diff
@@ -102,7 +103,7 @@ export SEP=$(subst /,\,/)
 DRUNTIME_PATH=..\..\druntime
 PHOBOS_PATH=..\..\phobos
 export DFLAGS=-I$(DRUNTIME_PATH)\import -I$(PHOBOS_PATH)
-export LIB=$(PHOBOS_PATH)
+export REQUIRED_ARGS=-L/FORCE:MULTIPLE
```

Comment:
I guess dmd-testsuite hasn't worked for Win32/MinGW either, as the chosen C compiler was dmc. I know that this is all some hacky business, that's why AppVeyor currently merges this branch before executing the tests.
I could introduce an `OS=win-msvc` for a nicer solution, but I'll have to check what's depending on the `OS` environment variable (hopefully no test itself).